src/redux/books/books.js

```diff
@@ -0,0 +1,49 @@
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+
+const addBook = (id, title, author) => ({
+  type: ADD_BOOK,
+  id,
+  title,
+  author,
+});
```

Comment:
[Optional] in the form for adding books, I suggest that you clear the content after dispatching the addBook action so that the user will have a fresh form to enter another book.